src/utils/eventLevels.js

```diff
@@ -8,12 +8,12 @@ export function endOfRange(dateRange, unit = 'day') {
   }
 }
 
-export function eventSegments(event, first, last, { startAccessor, endAccessor }) {
+export function eventSegments(event, first, last, { startAccessor, endAccessor }, range) {
   let slots = dates.diff(first, last, 'day')
   let start = dates.max(dates.startOf(get(event, startAccessor), 'day'), first);
   let end = dates.min(dates.ceil(get(event, endAccessor), 'day'), last)
 
-  let padding = dates.diff(first, start, 'day');
+  let padding = range.findIndex(x => x.getTime() === start.getTime());
```

Comment:
I'm not sure if this will work in all cases...is it certain that these will be exactly the same time? Perhaps you `dates.eq(x, start, 'day')` just to be safe?